From 360b14eca3618dd13b0502282ba1a171368b0d61 Mon Sep 17 00:00:00 2001
From: Connor Feeley <git@cfeeley.org>
Date: Mon, 19 Dec 2022 14:01:42 -0500
Subject: [PATCH 3/3] fix: don't bundle manually

---
 CMakeLists.txt | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5c014815..caabafd6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -260,18 +260,12 @@ if (${INPUTLEAP_BUILD_INSTALLER})
 # macOS app Bundle
 #
 if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-    set (CMAKE_INSTALL_RPATH "@loader_path/../Libraries;@loader_path/../Frameworks")
     set(INPUTLEAP_BUNDLE_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/dist/macos/bundle)
     set(INPUTLEAP_BUNDLE_DIR ${CMAKE_BINARY_DIR}/bundle)
     set(INPUTLEAP_BUNDLE_APP_DIR ${INPUTLEAP_BUNDLE_DIR}/Barrier.app)
     set(INPUTLEAP_BUNDLE_BINARY_DIR ${INPUTLEAP_BUNDLE_APP_DIR}/Contents/MacOS)
 
     configure_files(${INPUTLEAP_BUNDLE_SOURCE_DIR} ${INPUTLEAP_BUNDLE_DIR})
-
-    add_custom_target(Barrier_MacOS ALL
-                      bash build_dist.sh
-                      DEPENDS barrier barriers barrierc
-                      WORKING_DIRECTORY ${INPUTLEAP_BUNDLE_DIR})
 endif()
 
 #
-- 
2.38.1

